package.json

```diff
@@ -59,11 +60,12 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "1.1.1",
-    "@edx/frontend-build": "12.4.15",
+    "@edx/frontend-build": "12.9.0-alpha.1",
     "@testing-library/jest-dom": "5.11.9",
     "@testing-library/react": "11.2.7",
     "@testing-library/react-hooks": "3.7.0",
     "@testing-library/user-event": "13.5.0",
+    "@types/node": "^18.15.11",
```

Comment:
[curious/clarification] Will all consumers/MFEs need this `@types/node` dependency?